### PR TITLE
gr-zeromq: Remove VLA from last_endpoint()

### PR DESCRIPTION
**Pull request by btashton**
The zeromq code as a whole code repeats itself a lot, but changing that would mean creating some new base classes and doing quite a bit of restructuring that I am not very comfortable doing.   This should remove the variable length array that is not allowed by MSVC.

### CODE REVIEW

**Comment by btashton** [comment]
@gvanem This should resolve your issue, but I don't have a MSVC environment to test it on.

**Comment by gvanem** [comment]
@btashton Your diff looks okay (untested though).

File: gr-zeromq/lib/base_impl.cc [hunk 1]
```diff
@@ -57,8 +57,8 @@ namespace gr {
     std::string
     base_impl::last_endpoint()
     {
-      size_t addr_len = 256;
-      char addr[addr_len];
+      char addr[256];
```
Comment (mbr0wn) [inline gr-zeromq/lib/base_impl.cc]: Or, you `constexpr` the first line and keep it as-is.

**Comment by btashton** [comment]
A comment that was made here: https://github.com/gnuradio/gnuradio/commit/b76a95c52f812181605248f1f590a199e2e5d0e3#commitcomment-31490370

**Pull request merged**